odk1-src/getting-started.rst

```diff
@@ -1,90 +1,81 @@
 Getting Started With ODK
 =========================
 
-.. _using-odk:
+This document walks you through a very basic setup process,
+to get you familiar with using Open Data Kit.
 
-Using ODK
------------
+You will:
 
-In most cases, users of ODK:
+.. contents::
+ :local:
 
-- Create survey forms using `Build <https://build.opendatakit.org/>`_ or `XLSForm <http://xlsform.org/>`_
-- Upload forms to an Aggregate server
-- Load forms into Collect on an Android device
-- Use Collect to fill out forms with :term:`participants <participant>`
-- Upload survey data from Collect to Aggregate
-- Analyze or export data in Aggregate
-
-This requires:
-
-- :doc:`Installing Collect on a phone or other mobile device <collect-install>`
-- :ref:`Installing Aggregate on a server <installing-aggregate>`
-
-.. tip::
-
-  While this is the *typical* use pattern, it is not the only way to do things. For example:
-
-  - Forms can be created using tools other than Build or XLSForm
-  - Forms can be transfered directly to Collect, without using Aggregate
-
-  ODK is a very flexible set of tools, and organizations will find their own best practices for adopting it.
-
-.. _install-collect:
+.. _getting-started-install-collect:
 
 Install Collect
 ---------------------
 
-The easiest way to install the ODK Collect App is `to get it from the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
+The easiest way to install the Collect App is `to get it from the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
 
-For other installation options, see :doc:`collect-install`.
+.. seealso:: :doc:`collect-install`
 
-.. _installing-aggregate:
+.. _getting-started-install-aggregate:
 
-Install Aggregate
----------------------
+Install Aggregate (optional)
```

Comment:
If you don't install Aggregate, you can't do the things in this list (e.g., create a form with Build and upload it to Aggregate and so on. 

I think maybe we should have different levels? So Level 1 would be to install Collect, fill out a form, and see it on Aggregate. Then Level 2 would be to a form, upload it to a server, download it to Collect, fill it out, and see it on Aggregate.

The flaw in this plan is that it's not so easy to install Aggregate. And even with the VM it's not that fun to get Collect/Aggregate integration. So maybe we need to have a sandbox server that we delete data from nightly?